test/wolfhsm_cfg.h

```diff
@@ -30,12 +30,12 @@
 /* #define WOLFHSM_CFG_NO_CRYPTO */
 /* #define WOLFHSM_CFG_SHE_EXTENSION */
 
-#define WOLFHSM_CFG_COMM_DATA_LEN 1152
+#define WOLFHSM_CFG_COMM_DATA_LEN 1280
 
 
 #define WOLFHSM_CFG_NVM_OBJECT_COUNT 30
 #define WOLFHSM_CFG_SERVER_KEYCACHE_COUNT 10
-#define WOLFHSM_CFG_SERVER_KEYCACHE_SIZE 1024
+#define WOLFHSM_CFG_SERVER_KEYCACHE_SIZE 1280
```

Comment:
perhaps explicitly set to `WOLFHSM_CFG_COMM_DATA_LEN`?

Reply:
Well, the magic number is more like 1190-ish plus some headers and junk to support passing an RSA4096 DER key.  So, we should probably set this to an actual number and then make sure COMM_DATA_LEN is larger than that plus the packet headers.  Ok to leave this way for now.